scripts/Muon/GUI/Common/plot_widget/base_pane/base_pane_model.py

```diff
@@ -0,0 +1,46 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+
+class BasePaneModel(object):
+
+    def __init__(self, context):
+        self.context = context
+
+    @staticmethod
+    def _generate_run_indices(workspace_list):
+        indices = [0 for _ in workspace_list]
+        return indices
```

Comment:
This would be a faster operation
```suggestion
        return [0] * len(workspace_list)
```